media/mathplayground/global.css

```diff
@@ -1,63 +1,236 @@
 html, body {
-	position: relative;
-	width: 100%;
-	height: 100%;
+    position: relative;
+    width: 100%;
+    height: 100%;
 }
 
 body {
-	color: #333;
-	margin: 0;
-	padding: 8px;
-	box-sizing: border-box;
-	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+    color: #333;
+    margin: 0;
+    padding: 8px;
+    box-sizing: border-box;
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
 }
 
 a {
-	color: rgb(0,100,200);
-	text-decoration: none;
+    color: rgb(0,100,200);
+    text-decoration: none;
 }
 
 a:hover {
-	text-decoration: underline;
+    text-decoration: underline;
 }
 
 a:visited {
-	color: rgb(0,80,160);
+    color: rgb(0,80,160);
 }
 
-label {
-	display: block;
+article {
+    color: white;
+    background-color: rgba(0.5, 0.5, 0.5, 0.3);
+    padding: 5px;
+    font-family: Cambria, Cochin, Georgia, Times, "Times New Roman", serif;
 }
 
-input, button {
-	font-family: inherit;
-	font-size: inherit;
-	-webkit-padding: 0.4em 0;
-	padding: 0.4em;
-	margin: 0 0 0.5em 0;
-	box-sizing: border-box;
-	border: 1px solid #ccc;
-	border-radius: 2px;
+.box-1 {
+    text-align: right;
+    grid-column: 1 / 2;
+    color: white;
+    vertical-align: middle;
 }
 
-input:disabled {
-	color: #ccc;
+.box-2 {
+    grid-column-start: 2;
+    grid-column-end: 4;
+}
+
+.box-3 {
+    color: white;
+    vertical-align: middle;
+    text-align: center;
+    grid-column: 2;
+}
+.box-4 {
+    color: white;
+    vertical-align: middle;
+    text-align: left;
+    grid-column: 3;
+}
+
+.boxItem {
+    flex: auto;
+    background-color: rgba(33, 33, 33, 0.8);
+    border: 1px solid black;
+    border-radius: 1rem;
+}
+
+.boxItem button {
+    background-color: transparent;
+    color: whitesmoke;
+    border: none;
+}
+
+.box-title {
+    display: flex;
+    justify-content: space-between;
+    color: whitesmoke;
+    padding: 0.5em;
 }
 
 button {
-	color: #333;
-	background-color: #f4f4f4;
-	outline: none;
+    color: #333;
+    background-color: #f4f4f4;
+    outline: none;
+}
+
+
+
+button:active {
+    color: gray;
 }
 
 button:disabled {
-	color: #999;
+    color: #999;
+}
+
+button:focus {
+    border-color: #666;
+}
+
+button:hover {
+    color: white;
 }
 
 button:not(:disabled):active {
-	background-color: #ddd;
+    background-color: #ddd;
 }
 
-button:focus {
-	border-color: #666;
+button, input {
+    font-family: inherit;
+    font-size: inherit;
+    -webkit-padding: 0.4em 0;
+    padding: 0.4em;
+    margin: 0 0 0.5em 0;
+    box-sizing: border-box;
+    border: 1px solid #ccc;
+    border-radius: 2px;
```

Comment:
This kind of intricate style stuff is really in Bootstrap's domain, so let's see if we can remove some of these. Otherwise, this stuff will definitely conflict with bootstrap's buttons and inputs. From first glance, I think we can just remove this entire CSS clause.

Reply:
Bootstrap isn't currently implemented as far as I can tell. But I can add bootstrap if you like. Makes sense for a refactor anyway.

Reply:
Bootstrap is included here in the main template: https://github.com/ccnmtl/mathplayground/blob/main/mathplayground/templates/index.html#L12